test/mode/modeNormal.test.ts

```diff
@@ -1402,7 +1402,7 @@ suite('Mode Normal', () => {
     start: [
       '//    We choose to write a vim extension, not because it is easy, but because it is hard|.',
     ],
-    keysPressed: 'Vgq',
+    keysPressed: 'gq$',
```

Comment:
I think we should both cover `Vgq` and `gp$`.

Reply:
I also want to cover the "Vgq". But I don't have enough time to work on the bug of `V`. Current implementation of `V` is just just kind of workaround and it's result is not like what we expected. If we insist on testing this `Vgq`, the more important thing will be postponed.

Reply:
I think you can keep previous tests, declaring the wrong behaviors with comment/explanation, and create your new ones.
Hope other members would not oppose.